plenum/common/messages/fields.py

```diff
@@ -133,6 +134,25 @@ def _specific_validation(self, val):
             return '{} is longer than {} symbols'.format(val, self._max_length)
 
 
+class DatetimeStringField(FieldBase):
+    _base_types = (str,)
+    _exception_values = []
+
+    def __init__(self, exception_values: []=None, **kwargs):
```

Comment:
change type hint and default value to something like
exceptional_values: Iterable[str]=[]
You can find Iterable in "typing" standard module

assignment operation change to something like
self._exception_values = exception_values if exception_values is not None else []
because it could cause error in the following "if" statement
"if val not in self._exception_values"
since NonType is not iterable

Reply:
done